src/Entity/Package.php

```diff
@@ -168,11 +175,10 @@ public function getActivities()
     /**
      * @param Activity[] $activities
      *
-     * @return $this
+     * @return Package
      */
     public function setActivities($activities)
```

Comment:
I assume we need an array here, then we can also require users to insert an array:

````
public function setActivities(array $activities)
````